astrovascpy/bloodflow.py

```diff
@@ -398,10 +398,11 @@ def boundary_flows_A_based(
         boundary_nodes_mask = (degrees == 1) & cc_mask
         graph_1 = graph.node_properties.loc[boundary_nodes_mask]
 
-        boundary_flows = np.zeros(shape=graph.n_nodes)  # initialize
+        boundary_flows = np.zeros(shape=graph.n_nodes, dtype=np.float64)  # initialize
```

Comment:
```suggestion
        boundary_flows = np.zeros(shape=graph.n_nodes, dtype=np.float64)  # initialize
```
AFAIU, `np.float64`  is already the default.
https://numpy.org/doc/stable/reference/generated/numpy.zeros.html

Reply:
True. I can remove it